scripts/jenkins_regress.sh

```diff
@@ -18,7 +18,7 @@ exit_code_decorator(){
 gpg --quiet --batch --yes --decrypt --passphrase="$GPG_KEY" --output profile.properties scripts/profile.properties.gpg
 
 exit_code_decorator "mvn clean compile"
-exit_code_decorator "mvn -Dgpg.skip -DtagsToExclude=com.snowflake.snowpark.PerfTest -e scoverage:report"
+exit_code_decorator "mvn -Dgpg.skip -DtagsToExclude=com.snowflake.snowpark.PerfTest -e scoverage:report -P test-coverage"
```

Comment:
In the test coverage test, we have to downgrade scala to 2.12.15 since the scoverage maven plugin doesn't support Scala 2.12.18